packages/p2p/src/components/buy-sell/buy-sell-row.jsx

```diff
@@ -51,7 +51,7 @@ const BuySellRow = ({ row: advert }) => {
     const is_my_advert = advert.advertiser_details.id === general_store.advertiser_id;
     const is_buy_advert = counterparty_type === buy_sell.BUY;
     const { name: advertiser_name, rating_average, rating_count } = advert.advertiser_details;
-    const rating_average_decimal = rating_average ? Number(rating_average.toFixed(1)) : null;
+    const rating_average_decimal = rating_average ? Number(rating_average).toFixed(1) : null;
```

Comment:
this won't change the star rating rate right?

Reply:
No, it won't. This is just to make sure to convert the `rating_average` to number first then round the decimal. While, previous code can possibly throw error if `rating_average` is not a number.